### PR TITLE
1739 lobby ui

### LINKED ISSUE #1739: Retro Frontend: Minimum-Viable Retro Lobby
## Issue - Enhancement

This will create the lobby front-end experience for Retro Meetings.

### Acceptance Criteria (optional)
- The Retro Lobby matches the wireframes
- Team Members can see the presence of their teammates
- Team Members can see the Retro outline on the left-hand side
- The navigation is feature-flagged (behind a user-token feature flag) such that the code path is dead and eliminated in production (can probably handle this by feature flagging the “Start Retrospective” button)
- A Team Member can click “Start Retrospective” in order to become the facilitator and begin the Retro Meeting

- **Estimated effort:** 13 points ([see CONTRIBUTING.md](https://github.com/ParabolInc/action/blob/master/CONTRIBUTING.md#points-and-sizes))

### PR DESCRIPTION
**Pull request by mattkrick**
fixes #1739 

Deviation from AC: still keeping this being a build feature flag. will move to a user-based feature flag once it's safe for users to test! 
- [ ] Turn on retro feature flag, Go to team dash, click retro button, you go to a lobby that looks the same as the action lobby

### CODE REVIEW

**Comment by jordanh** [comment]
Reviewing this now

File: src/universal/components/TaskEditor/TaskEditor.js [hunk 1]
```diff
@@ -174,6 +174,8 @@ class TaskEditor extends Component {
       styles.root,
       noText && styles.rootNoText
     );
+    const placeholder = 'Describe what “Done” looks like';
+    // const placeholder = 'What does “Done” look like?';
```
Comment (jordanh) [inline src/universal/components/TaskEditor/TaskEditor.js]: We can probably nuke this line

Comment (jordanh) [reply src/universal/components/TaskEditor/TaskEditor.js]: (the comment)

File: src/universal/components/NewMeetingSidebar.js [hunk 2]
```diff
@@ -2,104 +2,83 @@
...
   viewer: Viewer
 }
+
 const NewMeetingSidebar = (props: Props) => {
   const {
     localPhase,
     viewer
   } = props;
   const {team: {teamId, teamName}} = viewer;
+  const relativeLink = `/retro/${teamId}`;
```
Comment (jordanh) [inline src/universal/components/NewMeetingSidebar.js]: Ditto that here, this rel link could be abstracted if that's your intention

File: src/universal/components/NewMeetingLobby.js [hunk 3]
```diff
@@ -49,9 +76,9 @@ const NewMeetingLobby = (props: Props) => {
   const isPro = tier === PRO;
   const canStartMeeting = isPro || meetingsRemaining > 0;
   return (
-    <React.Fragment>
-      <TitleHeader>{'Retro Meeting Lobby'}</TitleHeader>
-      <TeamNameHeader>{`${teamName} Retro`}</TeamNameHeader>
+    <Lobby>
+      <LabelHeading>{'Retro Meeting Lobby'}</LabelHeading>
```
[the remark this answers is not in the document]
Comment (mattkrick) [reply src/universal/components/NewMeetingLobby.js]: fixed! there are still a lot of retro-specifics which i know i won't catch (like copy & compnents, etc). I'm OK with it for now, we'll fix that stuff up when we write the next meeting (governance, peer review, ???)

File: src/universal/components/NewMeetingLobby.js [hunk 4]
```diff
@@ -113,14 +115,14 @@ const NewMeetingLobby = (props: Props) => {
           buttonStyle="primary"
           colorPalette="warm"
           disabled={!canStartMeeting}
-          label={`Start ${meetingTypeToLabel[meetingType]} Meeting`}
+          label={`Start ${meetingLabel} Meeting`}
```
Comment (jordanh) [inline src/universal/components/NewMeetingLobby.js]: yeah man, now we're cooking with gas

File: src/universal/components/NewMeetingSidebar.js (hunk 2 above)
Comment (mattkrick) [reply src/universal/components/NewMeetingSidebar.js]: fixed

**Comment by jordanh** [comment]
# Test Cycle 1
## Status: _in progress_

## Tests
- [ ] Turn on retro feature flag, Go to team dash, click retro button, you go to a lobby that looks the same as the action lobby

## Test Notes

**Pull request merged**